Calibration/HcalAlCaRecoProducers/python/alcaIsoTracksProducerFilter_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+IsoTracksProdFilter = cms.EDFilter("AlCaIsoTracksProducerFilter",
+                                   TriggerEventLabel = cms.InputTag("hltTriggerSummaryAOD","","HLT"),
+                                   TriggerResultLabel= cms.InputTag("TriggerResults","","HLT"),
+                                   Triggers          = cms.vstring("HLT_IsoTrackHB","HLT_IsoTrackHE"),
```

Comment:
Are these triggers part of 2017 HLT menu already? What rate has been agreed on? It would be good to have some pointers related to the proposal of these triggers.